### DISCUSSION TITLE
Delete account stats in front-end side

### DISCUSSION THREAD

**Discussion by adrianomic** [mastodon/mastodon#17614] [General]
In app/javascript/mastodon/reducers/accounts.js

```javascript
const normalizeAccount = (state, account) => {
  account = { ...account };

  delete account.followers_count;
  delete account.following_count;
  delete account.statuses_count;

  return state.set(account.id, fromJS(account));
};
```

This code is from current version.
I see you don't import followers_count, following_count and statuses_count when import accounts.
Can you please give me the reason why you're doing that?

Thanks

**Comment by Gargron** [comment]
The counts are stored in a separate reducer to avoid causing unnecessary re-renders in places where accounts are displayed without the counts.
